File: 09-how-many-calories/main.cpp
/*
  Ernest Izdebski, 2022-09-07

  A program that asks the user to input how many cookies he or she actually ate
  and then reports how many total calories were consumed. Assumes a bag of
  cookies holds 40 cookies, and an individual cookie is equivalent to 300.
*/

#include <iostream>
using namespace std;


const int calories_per_cookie = 300;

int main() {

  // the program requirements specify:
  //   Write a program that asks the user to input how many cookies he or she actually ate and then reports how many total calories were  consumed.
  // it doesn't say anything about using the 'Bag of cookies' information,
  // so I didn't

  // define & populate variables
  int cookies_consumed;
  cout << "how many Cookies did you Eat? ";
  cin >> cookies_consumed;

  // perform output logic
  cout << "Total Calories consumed: " << calories_per_cookie * cookies_consumed << endl;

}

File: 09-how-many-calories/src/main.rs
mod ernie;

// this variable is unused
// const COOKIES_PER_BAG: i32 = 40;
const CALORIES_PER_COOKIE: f32 = 300.0;

fn main() {
    let cookies_consumed: f32 = ernie::input("Input total cookies consumed: ");
    println!("Total calories: {}", cookies_consumed * CALORIES_PER_COOKIE);
}
